neuralop/layers/mlp.py

```diff
@@ -45,6 +45,8 @@ def __init__(
             else None
         )
 
+        if n_dim > 3:
+            n_dim = 3
```

Comment:
Can we do a quick timing comparison on GPU, using 3D conv vs always 1D Conv + reshaping? 
Since the inputs are C-ordered and continguous, it might not make a (big) difference, in which case it will be cleaner to always use e.g Conv1D or Conv2D